spring-security-cas-extension/src/main/java/com/kakawait/spring/security/cas/client/Cas.java

```diff
@@ -0,0 +1,23 @@
+package com.kakawait.spring.security.cas.client;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.lang.annotation.*;
+
+/**
+ * @author Jonathan Coueraud

```

Comment:
I understand that you want to become famous but I think only one `@author` comment is enough :rofl: